awx/main/utils/formatters.py

```diff
@@ -237,6 +237,13 @@ def get_extra_fields(self, record):
                 fields[log_name] = getattr(settings, setting_name, None)
             elif log_name == 'type':
                 fields[log_name] = 'other'
+
+        uuid = (
+            getattr(settings, 'LOG_AGGREGATOR_TOWER_UUID', None) or
```

Comment:
Do we want to give users the ability to, effectively, overwrite their identifier?